token-client/src/main/java/com/sap/cloud/security/xsuaa/client/XsuaaDefaultEndpoints.java

```diff
@@ -36,6 +36,16 @@ public class XsuaaDefaultEndpoints implements OAuth2ServiceEndpointsProvider {
 	 *            - the cert URI of XSUAA. It is required in case of X.509
 	 *            certificate based authentication.
 	 */
+	public XsuaaDefaultEndpoints(@Nonnull URI baseUri, @Nullable URI certUri) {
```

Comment:
this is out of scope of this PR and I don't see you using the added constructor in the  basic auth sample either, please remove it.